src/main/java/com/linkedin/kmf/apps/configs/MirrorPipelineMonitorConfig.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.apps.configs;
+
+public class MirrorPipelineMonitorConfig {
+
+  public static final String CONSUME_SERVICE_CONFIG = "consume.props";
```

Comment:
Agreed. So consume_service_config, produce_service_config, topic_management_service_config will not be defined while zookeeper_connect_list_config will be defined.